integration/clean/clean_ssm_parameter/clean_ssm_parameter.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	"github.com/aws/aws-sdk-go-v2/service/ssm/types"
+	"log"
+	"time"
+)
+
+// delete ssm parameter older than 60 days
+const keepDurationSSMParameter = -1 * time.Hour * 24 * 14
+const parameterSearchKey = "Name"
+const parameterSearchValue = "AmazonCloudWatch"
+const parameterSearchOption = "Contains"
+
+func main() {
+	err := cleanSSMParameter()
+	if err != nil {
+		log.Fatalf("errors cleaning %v", err)
+	}
+}
+
+func cleanSSMParameter() error {
+	log.Print("Begin to clean ssm parm")
+
+	expirationSSMParameter := time.Now().UTC().Add(keepDurationSSMParameter)
+
+	cxt := context.Background()
+	ssmClient, err := getSSMClient(cxt)
+	if err != nil {
+		return err
+	}
+
+	ssmParameter, err := getSSMParameter(cxt, ssmClient, expirationSSMParameter)
+	if err != nil || len(ssmParameter) == 0 {
+		return err
+	}
```

Comment:
If the length of ssmParameter is zero, err would be nil I think. I think it's probably fine, but not sure if that was intentional

Reply:
Yes if the len is 0 return nil err